Wikipedia/Code/WMFLanguagesViewController.m

```diff
@@ -480,7 +480,7 @@ - (void)addLanguageButtonTapped {
 - (void)languagesController:(WMFLanguagesViewController *)controller didSelectLanguage:(MWKLanguageLink *)language {
     [MWKDataStore.shared.languageLinkController appendPreferredLanguage:language];
     [self reloadDataSections];
-    [controller dismissViewControllerAnimated:YES completion:NULL];
+    [controller dismissViewControllerAnimated:YES completion:self.userDismissalCompletionBlock];
```

Comment:
I found that this caused the completion to be called after the user tapped a language from the "Add another language" screen. It just felt like a step too soon, because the `WMFPreferredLanguagesViewController` was still on screen when `finishResumingApp` kicked off, potentially causing whatever followup modals from that sequence to present on top of that screen. I feel like we should keep this as NULL.

Reply:
I put it back to NULL in this case and updated the method comment in the header.